website/orders/models.py

```diff
@@ -119,6 +130,22 @@ class Product(models.Model):
         "bound to the venue. Empty means no limit.",
     )
 
+    scannable = models.BooleanField(default=False, help_text="Whether or not this product can be scanned.")
+    barcode = models.CharField(null=True, blank=True, max_length=13, help_text="Either an EAN-8 or EAN-13 barcode.")
```

Comment:
Maybe add a validator for 8 digits or 13 digits. Would a positiveintegerfield work better maybe?

Reply:
Also, let's make this unique!

Reply:
A validator is a good idea, I will also change it to a PositiveIntegerField.